wordpress-mobile/WordPress-iOS
# This Week widget: update table separator style

Ref #13044

This updates the _This Week_ widget table separators to be more in line with the system widget style.

---
**NOTES:**

There was an issue with the separator not appearing sometimes, as noted here https://github.com/wordpress-mobile/WordPress-iOS/pull/13148#pullrequestreview-335295249. I believe this change resolves that, but do let me know if you see it happen.

As noted on https://github.com/wordpress-mobile/WordPress-iOS/pull/13148, there are still some outstanding UI issues, which will be addressed in a future PR:
- The footer jumps when expanding the widget.
- When the expanded widget is refreshed (ex: scrolling away from then back to Today):
  - The first two rows show the empty view while being refreshed.
  - The footer jumps.

---
**To test:**

- Go to Stats > Widgets > `Use this site`.
- Add the `WordPress This Week` widget to your Today view.
- Verify the table separator line looks more like that on system widgets (in these examples, Stocks).


**Light Mode:**

This Week | Stocks
--------|-------
![this_week_light](https://user-images.githubusercontent.com/1816888/72194415-ccc38f00-33ca-11ea-9e0f-d14e6d0802b4.jpeg)        |       ![stock_light](https://user-images.githubusercontent.com/1816888/72194431-d3ea9d00-33ca-11ea-8c01-d3928a5c70aa.jpeg)


**Dark Mode:**

This Week | Stocks
--------|-------
![this_week_dark](https://user-images.githubusercontent.com/1816888/72194512-18763880-33cb-11ea-87ce-f81a6e4b58a0.jpeg)        |       ![stock_dark](https://user-images.githubusercontent.com/1816888/72194529-24fa9100-33cb-11ea-8422-be502894d8cf.jpeg)


---
PR submission checklist:
